gradle.properties

```diff
@@ -110,7 +110,7 @@ artifactRedirecting.androidx.compose.material.version=1.7.0-beta03
 artifactRedirecting.androidx.collection.version=1.4.0
```

Comment:
Branch `origin/integration-snap/612689d2322/to-integration` seems missing? It needs to be merged at the same time as `to-jb-main` is merged. I can help to create it.

Reply:
It's here, I'll merge it right after this